### PR TITLE
Release/1.5.0

### PR DESCRIPTION
**Pull request by wallentx**

### CODE REVIEW

File: chia/daemon/server.py [hunk 1]
```diff
@@ -244,7 +244,6 @@ async def incoming_connection(self, request):
                             self.log.error(f"Unexpected exception trying to send to websocket: {e} {tb}")
                             self.remove_connection(socket)
                             await socket.close()
-                            break
```
Comment (hoffmang9) [inline chia/daemon/server.py:247]: Is this a regression?

Comment (wjblanke) [reply chia/daemon/server.py:247]: ya from the websocket move. this fixes it so less likely to close everything

Comment (cmmarslender) [reply chia/daemon/server.py]: related PR https://github.com/Chia-Network/chia-blockchain/pull/12241

**Review by altendky — CHANGES_REQUESTED** [review]
So it's on the record, this is being replaced by CI efforts on https://github.com/Chia-Network/chia-blockchain/tree/long_lived/post_1.5.0 from which a will be processed into `main`.  This PR is not expected to be used anymore as far as I know.

**Pull request closed without merging**